raiden/messages/metadata.py

```diff
@@ -79,6 +79,7 @@ class Metadata:
     # inititally create the Metadata object as part of a message - this is the case when we are
     # the initiator of a transfer.
     original_data: Optional[Any] = None
+    encrypted_secret: Optional[EncryptedSecret] = field(default=None, repr=False)
```

Comment:
The class needs the same `class Meta` attributes set as the `RoutesMetadata` - so that it disappears from serialisation if it is `None`